src/status_im/navigation/screens.cljs

```diff
@@ -553,11 +554,11 @@
 
     {:name      :screen/settings.import-seed-phrase
```

Comment:
nit: `:screen/settings.missing-keypair-import-seed-phrase` to make it consistent with the private key and make it unique (not to confuse to actual import seed-phrase)